src/components/views/rooms/EditMessageComposer.tsx

```diff
@@ -122,28 +122,25 @@ export function createEditContent(
 interface IEditMessageComposerProps extends MatrixClientProps {
     editState: EditorStateTransfer;
     className?: string;
+    context: React.ContextType<typeof RoomContext>;
```

Comment:
got to say, calling all these things just `context` isn't the clearest thing, especially in cases like this where there are actually two contexts in play. `roomContext` would be a better name.

Not a showstopper though.